README.md

```diff
@@ -19,23 +19,83 @@ Supports Redis 2.x, 3.x, 4.x, and 5.x
 ```
 
 
-### Prometheus Configuration
+### Basic Prometheus Configuration
 
 Add a block to the `scrape_configs` of your prometheus.yml config file:
 
 ```yaml
 scrape_configs:
+  - job_name: redis_exporter
+    static_configs:
+    - targets: ['<<REDIS-EXPORTER-HOSTNAME>>:9121']
+```
+
+and adjust the host name accordingly.
 
-...
+### Prometheus Configuration to Scrape Several Hosts
 
-- job_name: redis_exporter
-  static_configs:
-  - targets: ['localhost:9121']
+Run the exporter with the command line flag `--redis.addr=` so it won't try to scrape 
+the local instance when the `/metrics` endpoint is hit.
+
+```yaml
 
-...
+scrape_configs:
+  ## config for the multiple redis targets that the exporter will scrape
+  - job_name: 'redis_exporter_targets'
+    static_configs:
+      - targets:
+        - redis://first-redis-host:6379
+        - redis://h:password@second-redis-host:6379
```

Comment:
This is not a safe way to pass a password.

Reply:
Agreed.

I would suggest that for `/scrape` targets that `redis://` is implied. It doesn't make sense to remotely scrape a unix socket. Only allowing `host:port`, would be a good thing to filter for in the input.

Reply:
I don't think we can simply drop the scheme from the target as some instances are using SSL and the `rediss://` scheme is used for that.

But I agree, this is not the best way to pass a password and I'm open for suggestions on how to enable users to scrape password-protected instances.

Reply:
What we do elsewhere is have the password come from a config file.

Reply:
> What we do elsewhere is have the password come from a config file.

That's doable what what would that look like then someone wants to scrape several password protected instances use the `/scrape` endpoint?

Reply:
We have different modules in the snmp/blackbox exporter for that, which is basically another url parameter.

Reply:
I'm not sure I understand what you mean.

Reply:
https://github.com/prometheus/snmp_exporter/blob/master/generator/FORMAT.md

Reply:
Hmm, I see. So it looks like you use the module name to indicate to the exporter which auth profile to use?
I guess I could do something similar. Have a password file deployed with the exporter where each password is associated with an ID and then the prometheus server can send the password_id as a parameter.

Reply:
I would suggest having one password from the normal command line/env methods to start with and iterate on additional passwords. This way we can get this merged and have a good start on better functionality.

Reply:
Sounds good, let's start with that so this PR can move forward.